src/webusb.c

```diff
@@ -127,13 +149,34 @@ void webusb_handle_config_get(Ctrl_cfg_type key) {
     webusb_pending_config_share = key;
 }
 
-void webusb_handle_config_set(Ctrl_cfg_type key, uint8_t preset) {
+void webusb_handle_config_set(Ctrl_cfg_type key, uint8_t preset, uint8_t values[5]) {
     if (key > 4) return;
     webusb_pending_config_share = key;
-    if      (key == PROTOCOL)   config_set_protocol(preset);
-    else if (key == SENS_TOUCH) config_set_touch_sens(preset, false);
-    else if (key == SENS_MOUSE) config_set_mouse_sens(preset, false);
-    else if (key == DEADZONE)   config_set_deadzone(preset, false);
+    config_nvm_t config;
+    config_read(&config);
+    if (key == PROTOCOL) config_set_protocol(preset);
+    else if (key == SENS_TOUCH) {
+        config.sens_touch_value_1 = values[1];
```

Comment:
Add some comments about what the other parts of the array do
Why skip zero here
What happens to the other slots and their values (3 and 4) below